test/cpp/process_containment_linux_fake_ops.cpp

```
@@ -0,0 +1,839 @@
+};
+
+} // namespace
+
+const ProcessContainmentLinuxOps &
+process_lifetime_process_containment_linux_ops() noexcept {
+    return kFakeOps;
+}
+
+namespace testing {
+
+bool reset_process_containment_linux_fake() noexcept {
```

Cppcheck (2.21.0)

[style] The function 'reset_process_containment_linux_fake' is never used.

(unusedFunction)